### DISCUSSION TITLE
Cleanup when using "internal" flag

### DISCUSSION THREAD

**Discussion by gec75** [wills106/homeassistant-solax-modbus#879] [General]
@wills106
I saw you have already used the "internal" flag 👍 
But there is room for more cleanup :) ... you don't need any HA specific attributes anymore; just keep "key" and Modbus specific, but you don't need name, icon, native_unit_of_measurement, etc...

**Comment by wills106** [comment]
I think I removed all of the icons for Internal within the SolaX plugin. But I didn't have time yesterday to check if any other values are required or not. I'll have a proper look at the Solinteg plugin.

Would be good to see if there is a way to automatically purge dead entities preventing #878

**Reply by gec75** [reply]
I just had a quick look, and there are some "name", "native...", "device_class", "entity_category", etc.
There is no rush, I just wanted to mention.
